src/components/ToDoComponent/SettingGoals/CreateGoals.tsx

```diff
@@ -7,36 +7,76 @@ import {
   StyleSheet,
   TouchableOpacity,
   Image,
-  FlatList
+  FlatList,
+  Alert,
 } from 'react-native';
 import DeadLineDatePicker from './SettingGoalsComponents/DeadLineDatePicker';
 import { AuthContext } from '../../../context/AuthContext';
-import { getAllUsers } from '../../../screens/AuthScreen/Util';
+import { PostGoal, getAllUsers } from '../../../screens/AuthScreen/Util';
 
 const MainScreen = ({ navigation }) => {
-  const [selectedMember, setSelectedMember] = React.useState('');
   const [titleText, setTitleText] = useState('');
   const [descriptionText, setDescriptionText] = useState('');
-  const [isDropDownSelected, setIsDropDownSelected] =useState(false)
+  const [isDropDownSelected, setIsDropDownSelected] = useState(false);
   const [searchQuery, setSearchQuery] = useState('');
-  const [allUsers,setAllUsers] = useState([]);
+  const [allUsers, setAllUsers] = useState([]);
+  const [selectedUser, setSelectedUser] = useState('');
   const [isLoading, setIsLoading] = useState(true);
   const { loggedInUserData } = useContext(AuthContext);
+  const [titleError, setTitleError] = useState('');
+  const [descriptionError, setDescriptionError] = useState('');
 
+  const selectDropDown = () => {
+    setIsDropDownSelected(!isDropDownSelected);
+  };
 
-
-  const selectDropDown =()=>{
-    setIsDropDownSelected(!isDropDownSelected)
-  }
-
-    useEffect(() => {
-        fetchData(); 
-    });
+  useEffect(() => {
```

Comment:
what do you want to do over here ? don't you think this will render again and again even if will swipe the cards ?